python/pyspark/sql/group.py

```diff
@@ -194,6 +194,28 @@ def pivot(self, pivot_col, values=None):
             jgd = self._jgd.pivot(pivot_col, values)
         return GroupedData(jgd, self.sql_ctx)
 
+    def apply(self, udf_obj):
```

Comment:
We may possible pass in non-vectorized udf.

Reply:
Done